cc.cubocore.CoreTime.yml

```diff
@@ -0,0 +1,40 @@
+app-id: cc.cubocore.CoreTime
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: coretime
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --share=network # Required to show weather info
+  - --socket=wayland
+  - --filesystem=host
+  - --socket=pulseaudio # Required for ringing alarm
```

Comment:
why does a "time related task manager" require access to host and sound?

Reply:
host access to have a single settings file so all other apps in the suite act same, and sound access for ringing alarms as written in the comments

Reply:
is this a blocker?

Reply:
> host access to have a single settings file so all other apps

why would that need whole host access? where are you storing these?

Reply:
at home/.config directory. just adding home dir access doesn't work, only host access works. Also if the alarm sound file in a drive other then home, app can't able to access them.

Reply:
This doesn't make sense, if it's `home/.config` it should need that permission only, why would it need the rest of host?